_rules/54621b.md

```diff
@@ -0,0 +1,103 @@
+---
+id: 54621b
+name: aria-invalid Identifies Input Error
+rule_type: atomic
+
+description: |
+  This rule checks that the `aria-invalid` attributed is used to identify input errors
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag-technique: ARIA21: # Using Aria-Invalid to Indicate An Error Field
```

Comment:
I think we do something like `aria11:state_property_processing`....

Reply:
But these are tests for WCAG techniques. Why would we change this to aria?